cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -453,6 +453,174 @@ describe("Timeline", () => {
             cy.get(".mx_RoomView_searchResultsPanel").percySnapshotElement("Highlighted search results");
         });
 
+        it("should find words in various languages and highlight them on search result", () => {
+            // "Test" in various languages
+            const stringAr = "اِمْتِحَان‎";
+            const stringEl = "εξέταση";
+            const stringHe = "מִבְחָן";
+            const stringHi = "आज़माइश";
```

Comment:
https://en.wiktionary.org/wiki/%E0%A4%86%E0%A4%9C%E0%A4%BC%E0%A4%AE%E0%A4%BE%E0%A4%87%E0%A4%B6#Hindi